virttest/utils_disk.py

```diff
@@ -259,6 +259,21 @@ def get_windows_disks_index(session, image_size):
     return disk_indexs
 
 
+def get_windows_disk_index_by_serial(session, serial_number):
```

Comment:
Suggest moving this to `virttest.utils_windows.drive`.

Reply:
Thanks, have update, done.